packages/components/src/tooltip/index.js

```diff
@@ -212,9 +259,12 @@ function Tooltip( props ) {
 		? getDisabledElement
 		: getRegularElement;
 
+	const placement = positionToPlacement( position );
+
 	const popoverData = {
+		anchorRef: childRef,
 		isOver,
-		position,
+		placement,
```

Comment:
Should we consider updating the APIs of `Tooltip` to match the ones of `Popover`? (i.e. adding the `placement` prop and deprecating the `position` prop)